valinta-tulos-service/src/main/scala/fi/vm/sade/valintatulosservice/VtsServletBase.scala

```diff
@@ -85,7 +85,7 @@ trait VtsServletBase extends ScalatraServlet with Logging with JacksonJsonSuppor
   }
 
   protected def checkJsonContentType() {
-    if (request.requestMethod == Post && request.contentType.forall(!_.contains("application/json"))) {
+    if (request.requestMethod == Post && request.body.nonEmpty && request.contentType.forall(!_.contains("application/json"))) {
```

Comment:
Onko tyhjä body varmasti OK?

Reply:
RFC:n mukaan [ilmeisesti joo](https://tools.ietf.org/html/rfc7230#section-3.3.2): "For example, a Content-Length header field is normally sent in a POST request even when the value is 0 (indicating an empty payload body). "

Jos oikein muistan, tässä oli ongelmana että esim. Swagger sooloilee taas headereiden kanssa ja jättää Content-Type -headerin pois tyhjän bodyn tapauksessa.